packages/grid/_modules_/grid/components/columnHeaders/ColumnHeaderMenuIcon.tsx

```diff
@@ -1,4 +1,6 @@
 import * as React from 'react';
+// @ts-ignore TODO: remove once the module is fixed
```

Comment:
So we can't forget to remove it.

```suggestion
// @ts-expect-error fixed in Material-UI v5, types definitions were added.
```